packages/core/src/components/graphs/bubble.ts

```diff
@@ -66,17 +69,27 @@ export class Bubble extends Scatter {
 			)
 			// We need `|| 1` here in case the user doesn't provide radius values in data
 			.attr("r", (d) => radiusScale(d[radiusMapsTo] || 1))
-			.attr("fill", (d) =>
-				this.model.getFillColor(d[groupMapsTo], d[domainIdentifier], d)
+			.attr(
+				"class",
+				(d) => noProvidedColorScale
+					? `dot
+					${this.model.getStrokeColorClass(d[groupMapsTo])} 
+					${this.model.getColorClass(d[groupMapsTo])}
```

Comment:
Can this be a `this.model.getColorClassNames(["fill", "stroke"], d[groupMapsTo])`?